src/pages/films.js

```diff
@@ -1,10 +1,29 @@
-import React from "react";
+import React, {Fragment} from "react";
 import { graphql } from "gatsby";
 
+
 export default function Films({ data }) {
+   
+    const films = data.hasura.tnff_films
+    // filter film by year
+    // gallery with film image and title
+    const filmData = films.map(film => {
+        const { film_img_url, title, id, release_date } = film;
+        if (film_img_url) {
+            return (
+                <div className="film_single" key={id}>
+                    <img src={` https://${film_img_url}`} alt="" />
```

Comment:
i've update the table to have the https:// shouldn't have to prefix it with the https anymore